packages/ui/__tests__/__snapshots__/index.test.ts.snap

```diff
@@ -2,32 +2,6 @@
 
 exports[`exports ui 1`] = `
 {
-  "Accordion": {
```

Comment:
@ryanjdiercks do you know why this is happening?

This component's snapshot looks different than the others, I think it's because it's a styled component and not a regular one, is the one called `List` that you are exporting.

Reply:
Resolved it